packages/augur-tools/src/flash/template-utils.ts

```diff
@@ -19,3 +20,19 @@ const findTemplate = (category: CategoryTemplate, hash: string): string => {
   template = category.templates.find(t => t.hash === hash);
   return template;
 };
+
+export const validateMarketTemplate = (title: string, templateInfo: string, outcomesString: string, longDescription: string): string => {
+  const extraInfoTemplate = JSON.parse(templateInfo) as ExtraInfoTemplate;
+  let outcomes = [];
+  if (outcomesString) {
+    outcomes = outcomesString.split(',').map(s => stringTo32ByteHex(s));
```

Comment:
```suggestion
    outcomes = outcomesString.split(',').map(stringTo32ByteHex);
```

Reply:
I'm not a super fan of that syntax, too shortcutty, but I'll make the change